CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added 
 
-- Add support for server-side live transcription via Amazon Transcribe and Amazon Transcribe Medical. Speaker utterances will be transcribed in real time and delivered as data messages to your application, allowing you to implement subtitles, sentiment analysis, and other workloads. See [the live transcription guide](https://docs.aws.amazon.com/chime/latest/dg/meeting-transcription.html) for more details.
+- Supports integration with Amazon Transcribe and Amazon Transcribe Medical for live transcription.  The Amazon Chime Service uses its active talker algorithm to select the top two active talkers, and sends their audio to Amazon Transcribe (or Amazon Transcribe Medical) in your AWS account.  User-attributed transcriptions are then sent directly to every meeting attendee via data messages.  Use transcriptions to overlay subtitles, build a transcript, or perform real-time content analysis.  For more information, visit [the live transcription guide](https://docs.aws.amazon.com/chime/latest/dg/meeting-transcription.html).
```

Comment:
nit: `transcription.  The Amazon Chime Service` 2 spaces

Reply:
Fixed